lib/prometheus.sh

```diff
@@ -96,7 +96,7 @@ _load_prometheus_operator_images_to_local_registry() {
 		updated_image="$(_trim_prometheus_operator_image "$img")"
 		$CTR_CMD pull "$img"
 		$CTR_CMD tag "$img" "localhost:5001/$updated_image"
-		$CTR_CMD push "localhost:5001/${updated_image}"
+		$CTR_CMD push "localhost:5001/${updated_image}" --tls-verify=false
```

Comment:
If this works, I suspect something else to to be the solution because the flag seem invalid. 

```
docker help 
...
      --tls                Use TLS; implied by --tlsverify
      --tlsverify          Use TLS and verify the remote

```

Are you using `docker` ?